packages/scandipwa/src/route/MenuPage/MenuPage.container.js

```diff
@@ -52,15 +51,11 @@ export class MenuPageContainer extends PureComponent {
         });
     }
 
-    componentDidUpdate() {
-        this.redirectIfNotOnMobile();
-    }
```

Comment:
Why not redirect on componentDidUpdate? 
If device status will change, then it won't redirect in this case. (mobile => tablet)

Reply:
when I test it only on componentDidUpdate() the page didn't redirect its page, and if the redirect doubled with componentDidMount() the page got crashed